packages/components/ng-ui-router-breadcrumb/src/ng-ui-router-breadcrumb.service.js

```diff
@@ -64,12 +64,14 @@ export default class BreadcrumbService {
             transition
               .injector(state.name)
               .getAsync('breadcrumbPrefix')
-              .then(({ name, url }) => {
-                this.breadcrumb.unshift({
-                  name,
-                  url,
-                  active: false,
-                });
+              .then((prefixs) => {
+                prefixs.forEach(({ name, url }, index, prefixArray) =>
```

Comment:
**nitpick(blocking):**If user sends `[A, B]` array, the current implementation will result in the breadcrumb
`B > A > Current state`

The received array must be reversed and added to breadcrumb to display the desired result

`A > B > Current state`